bk-nodes/bk_db.py

```diff
@@ -0,0 +1,241 @@
+
+import MySQLdb
+import config
+import dateutil.parser
+
+
+table_fields = {}
+table_fields_index ={}
+
+
+class BeekeeperDB():
+    def __init__ ( self , retries=60) :
+
+        if not config.mysql_host:
+            raise Exception("mysql_host is not defined")
+        
+        count = 0
+        while True:
+            try:
+                self.db=MySQLdb.connect(host=config.mysql_host,user=config.mysql_user,
+                  passwd=config.mysql_password,db=config.mysql_db)
+            except Exception as e: # pragma: no cover
+                if count > retries:
+                    raise
+                print(f'Could not connnect to database, error={e}, retry in 2 seconds', file=sys.stderr)
```

Comment:
you need to `import sys`

i saw this error when doing a `docker-compose up`

```
bk-nodes_1     |   File "/usr/src/app/bk_db.py", line 25, in __init__
bk-nodes_1     |     print(f'Could not connnect to database, error={e}, retry in 2 seconds', file=sys.stderr)
bk-nodes_1     | NameError: name 'sys' is not defined
```

Reply:
fixed